modin/pandas/test/test_series.py

```diff
@@ -509,7 +504,16 @@ def test___repr__(name, dt_index, data):
             "1/1/2000", periods=len(pandas_series.index), freq="T"
         )
         pandas_series.index = modin_series.index = index
-    assert repr(modin_series) == repr(pandas_series)
+
+    try:
+        assert repr(modin_series) == repr(pandas_series)
+    except Exception as e:
```

Comment:
We possibly want to catch only `AssertionError` here, we don't want to enter in this `except` block if `repr(modin_series)` arise some exception.
```suggestion
    except AssertionError as e:
```

Reply:
Wait, are you seriously suggesting to replace simple `if repr(a) != repr(b): do_stuff()` with this `try.. assert.. except AssertionError.. do_stuff()`?..

If anything I suggest rewriting this piece into smth like
```python
if get_current_backend() == 'BaseOnPython` and data == 'empty':
    # TODO: explain reasons
    assert modin_series.dtype == np.object
    assert pandas_series.dtype == np.float64
else:
    assert repr(modin_series) == repr(pandas_series)
```